File: src/main/java/com/distribution/modules/integral/entity/ProductTypeEntity.java
package com.distribution.modules.integral.entity;

import lombok.Data;

import java.io.Serializable;


/**
 * 积分兑换产品类型表
 *
 * @author huchunliang
 * @email davichi2009@gmail.com
 * @date 2018-07-01
 */
@Data
public class ProductTypeEntity implements Serializable {
    private static final long serialVersionUID = 1L;

    /**
     *
     */
    private String id;
    /**
     * 产品类型名称
     */
    private String prodName;
    /**
     * 结算周期
     */
    private String prodRate;
    /**
     * 产品描述
     */
    private String prodRemark;

    /**
     * 图标
     */
    private String prodImg;

    private String prodType;
}

File: src/main/java/com/distribution/modules/integral/entity/ProductTypeEntity.kt
package com.distribution.modules.integral.entity

import java.io.Serializable


/**
 * 积分兑换产品类型表
 *
 * @author huchunliang
 * @email davichi2009@gmail.com
 * @date 2018-07-01
 */

data class ProductTypeEntity(
        /**
         *
         */
        var id: String = "",

        /**
         * 产品类型名称
         */
        var prodName: String = "",

        /**
         * 结算周期
         */
        var prodRate: String = "",

        /**
         * 产品描述
         */
        var prodRemark: String = "",


        /**
         * 图标
         */
        var prodImg: String = "",


        var prodType: String = "",

        var exchangePercent: Double = 0.00
) : Serializable {
    companion object {
        private const val serialVersionUID = 1L
    }
}
